internal/impl/azure/output_blob_storage.go

```diff
@@ -198,22 +223,21 @@ func (a *azureBlobStorageWriter) WriteBatch(_ context.Context, msg message.Batch
 		if err != nil {
 			return fmt.Errorf("blob type interpolation error: %w", err)
 		}
-
-		c := a.client.GetContainerReference(containerStr)
-		b := c.GetBlobReference(pathStr)
-		if err = a.uploadBlob(b, blobTypeStr, p.AsBytes()); err != nil {
+		ctx := p.GetContext()
```

Comment:
I need to familiarise myself with how this context is used, but shouldn’t we use here the `ctx` parameter of the method?

Reply:
You are correct. I think I got used to retrieve the context from the message on custom processors and messed up here. Thanks!